src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/OverloadOperatorEqualsOnOverridingValueTypeEquals.cs

```diff
@@ -39,6 +39,7 @@ public override void Initialize(AnalysisContext analysisContext)
             {
                 var namedTypeSymbol = (INamedTypeSymbol)context.Symbol;
                 if (namedTypeSymbol.IsValueType &&
+                    !(namedTypeSymbol.IsRefLikeType && namedTypeSymbol.TypeKind == TypeKind.Struct) &&
```

Comment:
Since we already have the `IsValueType` check, can we just do this?

```suggestion
                    !namedTypeSymbol.IsRefLikeType &&
```

Reply:
Indeed I went for the stupid copy-paste... Sorry